utils/benchmark/setupForTests.sh

```diff
@@ -35,32 +42,28 @@ done
 
 # Install @ima scoped packages from local registry
 npm config set @ima:registry=$NPM_LOCAL_REGISTRY_URL
+npm config set legacy-peer-deps true
 
 # Update create-ima-app versions
-cd "$ROOT_DIR"
+cd "$ROOT_DIR_IMA"
 node utils/version/create-ima-app-versions.js
 # Link current create-ima-app version to global scope
 cd "$CREATE_IMA_APP_DIR"
 npm link
 
 # Setup app from example hello
-cd "$ROOT_DIR"
+cd "$ROOT_DIR_IMA"
 npx create-ima-app ima-app
 
-cd ./ima-app
+mv "$ROOT_DIR_IMA/ima-app" "$ROOT_DIR"
 
-npm run build
-# Add customized environment configuration
-mv server/config/environment.js server/config/environment.orig.js
-cp "$ROOT_DIR/utils/benchmark/app/environment.js" server/config/environment.js
-NODE_ENV=prod node server/server.js &
-IMA_SKELETON_SERVER_PID=$!
+cd "$ROOT_DIR_IMA_APP"
 
-sleep 7
+npm run build
 
-# Run test
-cd "$ROOT_DIR"
-node_modules/.bin/autocannon -c $PARALLEL_TEST_CONNECTIONS --no-progress "$TARGET_WEB_URL"
+# Run tests
+source "$ROOT_DIR_IMA/utils/benchmark/benchmarkTest.sh"
```

Comment:
The `benchmark` directory name is now very misleading about its contents, since these are more like create-ima-app tests.

Reply:
Directory renamed to `tests`.